### PR TITLE
Update IOutputMetadataResponse

### LINKED ISSUE #1884: [Bug]: Inconsistency in the output metadata response interface
### Issue description

The parameters of the `IOutputMetadataResponse` interface in `nodejs` are different from the response. Parameters like `included` or `spent` are not in the interface

Output metadata response
Captura de pantalla de 2024-01-25 11-23-03

`IOutputMetadataResponse` interface in the nodejs bindings
Captura de pantalla de 2024-01-25 11-23-15

### Version

latest changes of the `2.0` branch

### Expected behaviour

same interface as the response

### Actual behaviour

.

### Can the issue reliably be reproduced?

Yes

### Steps to reproduce the issue

log an output and see the metadata

### Errors

_No response_

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Update IOutputMetadataResponse to https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/tip48/tips/TIP-0048/openapi3-core.yaml

## Links to any relevant issues

Fixes #1884 

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: bindings/nodejs/lib/types/models/api/output-metadata-response.ts [hunk 1]
```diff
@@ -1,41 +1,69 @@
...
+ */
+export interface IOutputInclusionMetadata {
+    /**
+     * Slot in which the output was included.
+     */
+    slot: SlotIndex;
+    /**
+     * Transaction ID that created the output.
+     */
+    transactionId: BlockId;
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/types/models/api/output-metadata-response.ts]: Not BlockId

File: bindings/nodejs/lib/types/models/api/output-metadata-response.ts [hunk 2]
```diff
@@ -1,41 +1,69 @@
...
+     * Slot in which the output was spent.
      */
-    commitmentIdSpent?: HexEncodedString;
+    slot: SlotIndex;
     /**
-     * The transaction this output was spent with.
+     * Transaction ID that spent the output.
      */
-    transactionIdSpent?: HexEncodedString;
+    transactionId: BlockId;
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/types/models/api/output-metadata-response.ts]: Not BlockId

File: bindings/nodejs/lib/types/models/api/output-metadata-response.ts [hunk 3]
```diff
@@ -1,41 +1,69 @@
...
-     * Tells if the output is spent in a confirmed transaction or not.
+     * Metadata of the output if it is marked as spent in the ledger.
      */
-    isSpent: boolean;
+    spent?: IOutputConsumptionMetadata;
     /**
-     * The current latest commitment id for which the request was made.
+     * Latest commitment ID of the node.
      */
     latestCommitmentId: HexEncodedString;
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/types/models/api/output-metadata-response.ts]: Change to SlotCommitmentId?

**Review by thibault-martinez — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]
spend and consumed are the same, but i guess that could change. otherwise LGTM

**Pull request merged**